dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/BaseClasses.mo

```diff
@@ -1,19 +1,23 @@
 within Dynawo.Electrical.Controls.Voltage;
 
 /*
-* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* Copyright (c) 2021, RTE (http://www.rte-france.com)
 * See AUTHORS.txt
 * All rights reserved.
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, you can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 *
-* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
+* for power systems.
 */
 
-model TCL "Tap Changer Lock (TCL)"
-  /* Lock tap changers when the voltage level goes below a predefined threshold
+package BaseClasses
+  extends Icons.BasesPackage;
+
+partial model BaseTCL "Base model for TapChangerLocking"
```

Comment:
We can use the dev to go to BaseTapChangerBlocking. 
In addition, in the comment, I would suggest "Base model for Tap Changer Blocking (TCB)"

(apparently, from a quick search on the internet, blocking instead of locking is the good wording)

Reply:
done